sso updates (#1016)
neo4j/docs-aura: modules/ROOT/pages/security/single-sign-on.adoc

Reviewer: ```suggestion
```

:page-aliases: platform/security/single-sign-on.adoc
:page-role: aura-db-dedicated aura-db-business-critical aura-ds-enterprise

> Single Sign-On (SSO) enables organization owners and organization admins to use your organization’s identity provider (IdP) to authenticate users so they can access the Aura console and Aura instances.

Reviewer: ```suggestion
Single Sign-On (SSO) enables organization owners and organization admins to use their organization’s identity provider (IdP) to authenticate users so they can access the Aura console and Aura instances.
```

> There are two different ways to secure:

Reviewer: What does this mean? Are there two ways to secure the console and two ways to secure instances? Or can you set up SSO on two different levels, ie console and instance?

Author: the latter

* *Use as a log in for the Organization:* Login with SSO to the Aura console. Allows org admins to control how users log in when they are trying to access the organization.

* *Use as login method for instances within Projects in this Org:* The SSO log in will be on the instances. Impacts new database instances created within that project.

[IMPORTANT]
====
Single sign-on for instances only works for AuraDB Business Critical and AuraDB Virtual Private Cloud. 
It does not cover AuraDB Professional instances.

> ====

Reviewer: ```suggestion
```
This is already indicated by the labels. If you want to add it here as well, add it as a sentence, possible using *bold* for emphasis, to line 31.
Two admonitions after one another is not good UX.

Author: Thank you for that suggestion - great idea to put it as a sentence up top.

The purpose it so explain:

**SSO for console works for any tier
SSO for instances works for BC + VDC instances**

[IMPORTANT]
====
Existing customers with existing instances that want to turn on single sign-on, need to know it doesn't affect existing instances. 
It will only affect new instances that you create. 
A workaround is to clone those instances. 

> Once SSO is configured, clone those instances to a new instance, and the new instance will be protected by SSO.

Reviewer: ```suggestion
Note that SSO does *not* apply retroactively.
If you set it up, only *new* instances are affected.
To apply SSO to existing instances, you need to clone the instances to new instances, and thus work around this limitation.
```

.SSO configuration
[.shadow]

> image::security/sso/sso-configuration.png[A screenshot of the SSO configuration,320,240]

Reviewer: Maybe make this image bigger?

Author: will do!

This access then applies to *all newly created* instances the user has access to.

> *Use as login method for instances within Projects in this Org:* ensures users logging in with SSO have access to the database instances within the project.

Reviewer: ```suggestion
When used as login method for instances within Projects in this Org, it ensures that users logging in with SSO have access to the database instances within the project.
```
